### PR TITLE
 sys-fs/cryptmount: bump to version 5.3.1

### PR DESCRIPTION
**Pull request by ConiKost**
Closes: https://bugs.gentoo.org/628734
Closes: https://bugs.gentoo.org/641176
Package-Manager: Portage-2.3.62, Repoman-2.3.12
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>

@mattst88: Please have a look, I've completly rewritten that eBuild, so maybe it could be keeped?

### CODE REVIEW

File: sys-fs/cryptmount/cryptmount-5.3.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		$(use_enable mount delegation)
+		$(use_enable nls)
+		$(use_enable luks)
+		$(use_enable openssl openssl-compat)
+		$(use_with systemd)
+		$(use_enable udev libudev)
+
+	)
+
+	econf "${econf[@]}"
```
Comment (a17r) [inline sys-fs/cryptmount/cryptmount-5.3.1.ebuild]: there's something obviously wrong with this line. ;)

File: sys-fs/cryptmount/cryptmount-5.3.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	virtual/libiconv
+	virtual/libintl
+"
+
+DEPEND="
+	${RDEPEND}
+	sys-kernel/linux-headers
+"
+
+DOCS=(
```
Comment (a17r) [inline sys-fs/cryptmount/cryptmount-5.3.1.ebuild]: no need to expand this over multiple lines.

File: sys-fs/cryptmount/cryptmount-5.3.1.ebuild (hunk 1 above)
Comment (ConiKost) [reply sys-fs/cryptmount/cryptmount-5.3.1.ebuild]: Whoosps x) Should be fixed now. There is one problem open with USE="udev", will have a look.

File: sys-fs/cryptmount/cryptmount-5.3.1.ebuild (hunk 2 above)
Comment (ConiKost) [reply sys-fs/cryptmount/cryptmount-5.3.1.ebuild]: Ok.

File: sys-fs/cryptmount/cryptmount-5.3.1.ebuild (hunk 1 above)
Comment (ConiKost) [reply sys-fs/cryptmount/cryptmount-5.3.1.ebuild]: Udev is also now fixed.

**Pull request closed without merging**